File: JS/URI/1929.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

let j = 0;
let a = parseInt(lines[j].split(" ")[0]);
let b = parseInt(lines[j].split(" ")[1]);
let c = parseInt(lines[j].split(" ")[2]);
let d = parseInt(lines[j].split(" ")[3]);

if(a+b>c && a+c>b && b+c>a)
    console.log("S");
else if(b+c>d && b+d>c && c+d>b)
    console.log("S");
else if(a+c>d && a+d>c && c+d>a)
    console.log("S");
else if(a+b>d && b+d>a && a+d>b)
    console.log("S");
else
    console.log("N");

File: Python/URI/1929.py
a, b, c, d = map(int, input().split())

if (a + b > c and a + c > b and b + c > a):
    print("S")
elif (b + c > d and b + d > c and c + d > b):
    print("S")
elif (a + c > d and a + d > c and c + d > a):
    print("S")
elif (a + b > d and b + d > a and a + d > b):
    print("S")
else:
    print("N")
